roles/confluent.ksql/templates/ksql-server_log4j.properties.j2

```diff
@@ -44,13 +44,57 @@ log4j.appender.kafkaAppender.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
 log4j.appender.kafkaAppender.MaxFileSize={{ksql.kafka_appender_log_file_size}}
 log4j.appender.kafkaAppender.MaxBackupIndex={{ksql.kafka_appender_max_log_files}}
 log4j.appender.kafkaAppender.append=true
+{% if ksql_log_streaming_enabled | bool %}
+ksql.logging.processing.topic.auto.create: {{ksql_log_streaming_enabled | lower}}
```

Comment:
These need to get into ksql-server.properties (or whatever the filename is) not the log4j props right?

Reply:
You're right... this is what I get for editing late at night.